chroma-agent/tests/actions_plugins/test_agent_updates.py

```diff
@@ -67,32 +68,25 @@ def test_unconfigure_repo_no_file(self):
         self.assertEqual(agent_updates.unconfigure_repo(self.tmpRepo.name), agent_result_ok)
 
     def test_kernel_status(self):
-        def try_run(args):
-            if args == ["rpm", "-qR", "lustre-modules"]:
-                return """/bin/sh
-/bin/sh
-/bin/sh
-kernel = 2.6.32-358.18.1.el6
-rpmlib(CompressedFileNames) <= 3.0.4-1
-rpmlib(FileDigests) <= 4.6.0-1
-rpmlib(PayloadFilesHavePrefix) <= 4.0-1
-rpmlib(PayloadIsXz) <= 5.2-1
-"""
-            elif args == ["uname", "-r"]:
-                return "2.6.32-358.2.1.el6.x86_64\n"
-            elif args == ["rpm", "-q", "kernel"]:
-                return """kernel-2.6.32-358.2.1.el6.x86_64
-kernel-2.6.32-358.18.1.el6.x86_64
-"""
-
-        with patch('chroma_agent.lib.shell.AgentShell.try_run', side_effect=try_run):
+        def run(arg_list):
+            values = {("rpm", "-q", "--whatprovides", "kmod-lustre"):
+                      "kmod-lustre-1.2.3-1.el6.x86_64\n",
```

Comment:
It doesn't really matter.  It's a string that is part of a package release.  It has no meaning whatsoever.  It could say "blat".